tests/python/tank_test/tank_test_base.py

```diff
@@ -809,12 +902,25 @@ def _move_data(path):
             if os.path.exists(db_path):
                 print('Removing db %s' % db_path)
                 # Importing pdb allows the deletion of the sqlite db sometimes...
-                import pdb
+                import pdb # noqa
                 # try multiple times, waiting longer in between
                 for count in range(5):
                     try:
                         os.remove(db_path)
                         break
                     except WindowsError:
-                        time.sleep(count*2)
+                        time.sleep(count * 2)
             os.rename(path, backup_path)
+
+
+class TankTestSimple(TankTestBase):
+    """
+    Base class for running tests that need a scaffold similar to `TankTestBase` without
+    the pipeline configuration that is usually created. This gives a big speed boost
+    to many tests who don't even read what is on disk and therefore couldn't
+    care less about the scaffold.
+    """
+    def setUp(self, parameters=None):
```

Comment:
btw, not sure that using **kwargs would protect us from this, but here we're modifying a dictionary we don't own, so this could introduce side effects.

Reply:
We can't change that API at this point because other repos are using this signature and so are some clients.